src/Microsoft.AspNet.Mvc.Core/MvcCoreServiceCollectionExtensions.cs

```diff
@@ -78,24 +78,28 @@ internal static void AddMvcCoreServices(IServiceCollection services)
                 services,
                 ServiceDescriptor.Transient<IActionConstraintProvider, DefaultActionConstraintProvider>());
 
-            // Action Invoker
+            // Controller Factory
             //
             // This has a cache, so it needs to be a singleton
             services.TryAdd(ServiceDescriptor.Singleton<IControllerFactory, DefaultControllerFactory>());
+            // These will be cached by the controller factory
```

Comment:
Does this comment apply to both `DefaultControllerActivator` and `DefaultControllerPropertyActivator`? Break up with a blank line if not. Add a blank line above either way.